.github/workflows/kubernetes-integration-test.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           tags: thegreenwebfoundation/grid-intensity-exporter:integration-test
       - 
         name: Create kubernetes Kind Cluster
-        uses: helm/kind-action@v1.1.0
+        uses: helm/kind-action@v1.3.0
```

Comment:
The integration test was failing. 

This upgrades the Kubernetes version we test with from 1.19 to 1.24 and means the test will pass 🤷‍♂️